### ISSUE TITLE
selector-type-case

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#908]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Type selector may be in lowercase and uppercase. Need control this. 

Possible values for rule: "lower" and "upper".

> Which rule, if any, is this issue related to?

e.g. `selector-type-case`

> What CSS is needed to reproduce this issue?

e.g.

```css
// Valid
li > a {
    color: red 
}

// Valid
LI > A { 
    color: red 
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "selector-type-case": "upper"
  }
}
```

or 

```json
{
  "rules": {
    "selector-type-case": "lower"
  }
}
```

> Which version of stylelint are you using?

`5.0.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

PostCSS plugin

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

SCSS

> What did you expect to happen?

Throw error depending on the configuration.

> What actually happened (e.g. what warnings or errors you are getting)?

Nothing

**Comment by jeddy3** [comment]
Sounds good to me.

I'm going to mark this as "help wanted". I think this would make a great first rule for either yourself or someone else as everything need to get this rule done can be found in either [`color-hex-case`](https://github.com/stylelint/stylelint/tree/master/src/rules/color-hex-case) or [`selector-no-type`](https://github.com/stylelint/stylelint/tree/master/src/rules/selector-no-type) :)

**Comment by sh-waqar** [comment]
@evilebottnawi - If you have not started work on this. Kindly let me know. I would like to try it and to raise a PR. `First timer here` :)

**Comment by jeddy3** [comment]
Excellent :)

This rule will likely be included in `stylelint-config-standard`, and so it'd be a great rule as your first one.

Please don't hesitate to push up your PR at anytime if you need help with anything. Also, I recommend keeping an eye on [this PR](https://github.com/stylelint/stylelint/pull/916) as it contains lots of useful advice for writing a new rule.

**Comment by sh-waqar** [comment]
@jeddy3 Thanks :) I will try to raise a PR by today or tomorrow may be.

**Comment by evilebottnawi** [comment]
@sh-waqar not started. Let's PR :+1:

**Comment by davidtheclark** [comment]
This is done and released.
